presto-main/src/main/java/com/facebook/presto/testing/TestingAccessControlManager.java

```diff
@@ -336,7 +362,7 @@ private boolean shouldDenyPrivilege(String userName, String entityName, TestingP
         ADD_COLUMN, DROP_COLUMN, RENAME_COLUMN,
         CREATE_VIEW, DROP_VIEW, SELECT_VIEW,
         CREATE_VIEW_WITH_SELECT_TABLE, CREATE_VIEW_WITH_SELECT_VIEW,
-        SET_SESSION
+        SET_SESSION, SELECT_COLUMN
```

Comment:
This isn't related to `SET_SESSION`. Let's add it on the line with other permissions (`ADD_COLUMN`, etc.)

Reply:
Please move this above to be with the other column privileges.